.gitmodules

```diff
@@ -290,3 +290,6 @@
 	url = https://github.com/bouffalolab/bl_iot_sdk_tiny.git
         branch = main
         platforms = bouffalolab
+[submodule "third_party/libwebsockets/repo"]
+	path = third_party/libwebsockets/repo
+	url = https://github.com/warmcat/libwebsockets
```

Comment:
Should this have a nonempty "platforms" list?  I doubt any of the embedded platforms need this, right?

Reply:
I have added "platforms = darwin,linux" at the moment. If needed people can add more.